html-web-page/index.html

```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>HTML Web Page</title>
+  <style>
+    table,
+    th,
+    td {
+      border: 1px solid black;
+    }
+
+    td:not(:first-child) {
+      text-align: center;
+    }
+  </style>
+</head>
+<body>
+  <h1>Jelly of the Month Club</h1>
+  <h2>Make all your friends jelly of your stock of jelly!</h2>
+  <p>
+    Lorem ipsum dolor sit amet consectetur adipisicing elit. Adipisci vero quam doloremque voluptas maxime praesentium consequuntur corrupti nostrum molestiae ut eos illum veritatis autem expedita velit, enim suscipit minus harum?
+    Lorem ipsum dolor sit amet consectetur adipisicing elit. Sint ducimus reprehenderit voluptas beatae non dolorum facere at molestiae similique quod cumque saepe officiis ipsa sit mollitia, nulla perferendis corrupti labore?
+  </p>
+
+  <table>
+    <thead>
+      <tr>
+        <th></th>
+        <th>Basic Plan</th>
+        <th>Deluxe Plan</th>
+        <th>Jelly Giant Plan</th>
+      </tr>
+    </thead>
+    <tbody>
+   <tr>
+     <td>Three new curated jellies per month</td>
+     <th>&check;</th>
+     <th>&check;</th>
+     <th>&check;</th>
```

Comment:
These checkmarks are meant to be "table data", so they should be `td` elements. Make sure you change the others in your document as well 🤓